packages/cli/src/api/catalog.ts

```diff
@@ -23,17 +22,18 @@ const LOCALE_REPLACE_RE = /{locale}/g
 const LOCALE_SUFFIX_RE = /\{locale\}.*$/
 const PATHSEP = "/" // force posix everywhere
 
-type MessageOrigin = [string, number?]
+type MessageOrigin = [filename: string, line?: number]
 
 export type ExtractedMessageType = {
   message?: string
   origin?: MessageOrigin[]
-  extractedComments?: string[]
+  // todo: this actually not fulfilled by extractor, but actively used in PO formatter
+  //  need to check what was the difference from extractedComments and delete one
   comments?: string[]
```

Comment:
https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html

`comments` - from translator to developer
`extractedComments` - from developer to translator 

So extractor not related to `comments` at all. These comments should be preserved while merging catalogs.